### PR TITLE
[i18n-fr] Translate pipeline tutorial to French

### PR DESCRIPTION
**Pull request by NoB0**
# What does this PR do?

Translates the `pipeline_tutorial.md` file of the documentation to French.

Part of #21456

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

French speaking contributors.
Documentation: @stevhliu and @MKhalusova

### CODE REVIEW

File: docs/source/fr/pipeline_tutorial.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+
+Comme vous pouvez le voir, le modèle a déduit le texte et a également indiqué **quand** les différentes phrases ont été prononcées.
+
+Il y a beaucoup de paramètres disponibles pour chaque tâche, donc consultez la documentation de l'API de chaque tâche pour voir ce que vous pouvez paramétrer !
+Par exemple, le pipeline [`~transformers.AutomaticSpeechRecognitionPipeline`] a un paramètre `chunk_length_s` qui est utile quand on travaille sur des fichiers audio très longs (par exemple, sous-titrage de films entiers ou de vidéos d'une heure) qu'un modèle ne peut généralement pas gérer seul :
+
+```python
+>>> transcriber = pipeline(model="openai/whisper-large-v2", chunk_length_s=30, return_timestamps=True)
+>>> transcriber("https://huggingface.co/datasets/sanchit-gandhi/librispeech_long/resolve/main/audio.wav")
+{'text': " Chapter 16. I might have told you of the beginning of this liaison in a few lines, but I wanted you to see every step by which we came.  I, too, agree to whatever Marguerite wished, Marguerite to be unable to live apart from me. It was the day after the evening...
```
Comment (stevhliu) [inline docs/source/fr/pipeline_tutorial.md:152]:
```suggestion
{'text': " Chapter 16. I might have told you of the beginning of this liaison in a few lines, but I wanted you to see every step by which we came.  I, too, agree to whatever Marguerite wished, Marguerite to be unable to live apart from me. It was the day after the evening...}
```

**Review by stevhliu — APPROVED** [review]
The structure of the doc LGTM! Pinging @ArthurZucker for a quick review of the content :)

File: docs/source/fr/pipeline_tutorial.md (hunk 1 above)
Comment (NoB0) [reply docs/source/fr/pipeline_tutorial.md]: @stevhliu  Good catch! It appears that the brace is also missing in the English version of the documentation. Should I create an issue for that?

File: docs/source/fr/pipeline_tutorial.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+specific language governing permissions and limitations under the License.
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# Pipelines pour l'inférence
+
+Le [`pipeline`] facilite l'utilisation de n'importe quel modèle sur le [Hub](https://huggingface.co/models) pour l'inférence sur n'importe quelle tâche de langage, vision par ordinateur, parole et multimodale. Même si vous n'avez pas d'expérience avec une modalité spécifique ou si vous n'êtes pas familier avec le code associé aux modèles, vous pouvez toujours les utiliser pour l'inférence avec le [`pipeline`] ! Ce tutoriel vous apprend à :
```
Comment (ArthurZucker) [inline docs/source/fr/pipeline_tutorial.md:19]:
```suggestion
La [`pipeline`] facilite l'utilisation de n'importe quel modèle sur le [Hub](https://huggingface.co/models) pour l'inférence sur n'importe quelle tâche de langage, vision par ordinateur, parole et multimodale. Même si vous n'avez pas d'expérience avec une modalité spécifique ou si vous n'êtes pas familier avec le code associé aux modèles, vous pouvez toujours les utiliser pour l'inférence avec le [`pipeline`] ! Ce tutoriel vous apprend à :
```

File: docs/source/fr/pipeline_tutorial.md [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+
+  * Utiliser un [`pipeline`] pour l'inférence.
+  * Utiliser un tokenizer ou un modèle spécifique.
+  * Utiliser un [`pipeline`] pour des tâches audio, de vision et multimodales.
```
Comment (ArthurZucker) [inline docs/source/fr/pipeline_tutorial.md:23]: It is `une` for `la pipeline` not `le`

**Review by ArthurZucker — COMMENTED** [review]
Pipeline is a female gendered noun in french! Otherwise should be alright

**Comment by NoB0** [comment]
> Pipeline is a female gendered noun in french! Otherwise should be alright

According to [Larousse](https://www.larousse.fr/dictionnaires/francais/pipeline/61067) and [Le Petit Robert](https://dictionnaire.lerobert.com/definition/pipeline) dictionaries it is male gendered. Waiting for feedback before changing the gender.

File: docs/source/fr/pipeline_tutorial.md (hunk 1 above)
Comment (stevhliu) [reply docs/source/fr/pipeline_tutorial.md]: You can just add the missing brace for the English docs in this PR!

**Pull request closed without merging**